CMakeLists.txt

```diff
@@ -185,7 +185,7 @@ endif ()
 ###############################################
 ## Build the AWS IoT Device Client Executable #
 ###############################################
-add_executable(${DC_PROJECT_NAME} ${DC_SRC})
+add_executable(${DC_PROJECT_NAME} ${DC_SRC} source/util/ProxyUtils.cpp source/util/ProxyUtils.h)
```

Comment:
not sure how this change made it back, but don't forget to remove the changes to the top-level CMakeLists and test/CMakeLists

Reply:
Good catch! addressed